transformers/modeling_bert.py

```diff
@@ -1145,3 +1309,101 @@ def forward(self, input_ids, attention_mask=None, token_type_ids=None, position_
             outputs = (total_loss,) + outputs
 
         return outputs  # (loss), start_logits, end_logits, (hidden_states), (attentions)
+
+
+@add_start_docstrings("Bert encoder-decoder model for sequence generation.",
+                      BERT_START_DOCSTRING,
+                      BERT_INPUTS_DOCSTRING)
+class Bert2Rnd(BertPreTrainedModel):
```

Comment:
I agree. Do you think I could start with making something specific work and generalize later? It will be easier to do something general when I am not worried about having something functional + I am familiar with the codebase?